src/imp/tracer_imp.js

```diff
@@ -180,6 +181,9 @@ export default class TracerImp extends EventEmitter {
     startSpan(fields) {
         let spanImp = new SpanImp(this);
         spanImp.setFields(fields);
+        for (let key in this._options.default_span_tags) {
+            spanImp.addTags({[key]: this._options.default_span_tags[key]});
+        }
```

Comment:
`addTags` takes a map (and handles undefined correctly). 184-186 can be simply written as:

```javascript
spanImp.addTags(this._options.default_span_tags);
```